sentry_sdk/transport.py

```diff
@@ -172,7 +173,10 @@ def _send_request(
             }
         )
         response = self._pool.request(
-            "POST", str(self._auth.store_api_url), body=body, headers=headers
+            "POST",
+            str(self._auth.get_api_url(endpoint_type)),
```

Comment:
```suggestion
            self._auth.get_api_url(endpoint_type),
```

Why do we need `str` here? If we need some Py2/3 compatibility and `requests` limits what it takes, we could handle this in `get_api_url` itself, otherwise all callers would need to make adjustments (and this seems to be the only caller other than test code).